src/roslint/main.py

```diff
@@ -0,0 +1,56 @@
+# Software License Agreement (BSD License)
+#
+# copyright (c) 2014-2016 roslint contributors
+# all rights reserved
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above
+#    copyright notice, this list of conditions and the following
+#    disclaimer in the documentation and/or other materials provided
+#    with the distribution.
+#  * Neither the name of Willow Garage, Inc. nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+from catkin_tools.verbs.catkin_build import main as build_main
+from catkin_tools.verbs.catkin_build \
+    import prepare_arguments as build_prepare_arguments
+
+
+# use same prepare_arguments as catkin_build
+prepare_arguments = build_prepare_arguments
+
+
+def main(opts):
+    if opts.catkin_make_args is None:
+        opts.catkin_make_args = ['roslint']
+    else:
+        opts.catkin_make_args.append('roslint')
```

Comment:
```python
opts.catkin_make_args = opts.catkin_make_args or []
opts.catkin_make_args.append('roslint')
```
because all other falsey values are just as dangerous as __None__.  Consider __False__, 0, 0.0, {}, ( ,), "", set(), etc.